layouts/partials/head.html

```diff
@@ -15,12 +15,32 @@
     integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
     crossorigin="anonymous"
   />
+  <script
+    src="https://kit.fontawesome.com/83077f7eb5.js"
+    crossorigin="anonymous"
+  ></script>
```

Comment:
We already have a font awesome cdn above this one. Is there something unique about this one?

Reply:
No, I may have just duplicated it by accident

Reply:
I've seen some font awesome cdns only support certain icons, so if this one has something unique to it, it can stay. If you remove this cdn and the icons still work, then remove the new one.